src/App/App.js

```diff
@@ -52,13 +53,15 @@ class App extends Component {
     if (this.state.user.name) {
       return <button className='log-button' onClick={this.logOutUser}>Logout</button>
     } else if (this.state.hasLoginView) {
-      return 
+      return
```

Comment:
This function seems funky - consider how you could clean it up??

Reply:
Totally agree.  Maybe we don't need an `else if( )` or we can have the empty return as part of an `else`

Reply:
We will look at refactoring this!